lib/Folder/FolderManager.php

```diff
@@ -482,7 +482,7 @@ public function searchUsers(int $id, string $search = '', int $limit = 10, int $
 	/**
 	 * @param string $groupId
 	 * @param int $rootStorageId
-	 * @return list<array{folder_id: int, mount_point: string, permissions: int, quota: int, acl: bool, rootCacheEntry: ?ICacheEntry}>
+	 * @return list<array{folder_id: int, mount_point: string, permissions: int, quota: int, acl: bool, rootCacheEntry: ?CacheEntry}>
```

Comment:
Why is this change needed? Any way to avoid the OC ref?

Reply:
Not really, as we use a private class it specifies the private type is returned. So only way to really fix it is by only using public classes.